src/pages/saksbehandling/søknadsbehandling/opphold-i-utlandet/OppholdIUtlandet.tsx

```diff
@@ -62,130 +81,111 @@ const OppholdIUtlandet = (props: VilkårsvurderingBaseProps) => {
-                                        status: OppholdIUtlandetStatus.SkalVæreMerEnn90DagerIUtlandet,
-                                    }))
-                                }
-                                checked={formik.values.status === OppholdIUtlandetStatus.SkalVæreMerEnn90DagerIUtlandet}
-                            />
-                            <Radio
-                                label={intl.formatMessage({ id: 'radio.label.nei' })}
-                                name="status"
-                                onChange={() =>
-                                    formik.setValues((v) => ({
-                                        ...v,
-                                        status: OppholdIUtlandetStatus.SkalHoldeSegINorge,
-                                    }))
-                                }
-                                checked={formik.values.status === OppholdIUtlandetStatus.SkalHoldeSegINorge}
-                            />
-                            <Radio
-                                label={intl.formatMessage({ id: 'radio.label.uavklart' })}
-                                name="status"
-                                onChange={() =>
-                                    formik.setValues((v) => ({
-                                        ...v,
-                                        status: OppholdIUtlandetStatus.Uavklart,
-                                    }))
-                                }
-                                checked={formik.values.status === OppholdIUtlandetStatus.Uavklart}
-                            />
-                        </RadioGruppe>
+                        <Controller
+                            control={form.control}
+                            name="status"
+                            render={({ field, fieldState }) => (
+                                <RadioGruppe
+                                    legend={formatMessage('radio.oppholdIUtland.legend')}
+                                    feil={fieldState.error?.message}
+                                    onBlur={field.onBlur}
+                                >
+                                    <Radio
+                                        id={field.name}
+                                        radioRef={field.ref}
+                                        label={formatMessage('radio.label.ja')}
+                                        name="status"
+                                        onChange={() =>
+                                            field.onChange(OppholdIUtlandetStatus.SkalVæreMerEnn90DagerIUtlandet)
+                                        }
+                                        checked={field.value === OppholdIUtlandetStatus.SkalVæreMerEnn90DagerIUtlandet}
+                                    />
+                                    <Radio
+                                        label={formatMessage('radio.label.nei')}
+                                        name="status"
```

Comment:
Hvorfor har bare den første radio-button id og radioRef? Og hvorfor bruker vi field.name noen plasser og string andre? Har sett litt forskjellige varianter fra fil til fil.

Reply:
Første radio-button får id og ref fordi det er den som skal fokuseres dersom man trykker på valideringsfeilen i feiloppsummeringa. Det at det noen plasser brukes strenger for `name` er nok stort sett fordi jeg glemte å fikse det i denne slengen 😅